.changeset/healthy-snails-unite.md

```diff
@@ -0,0 +1,44 @@
+---
+"@salt-ds/theme": minor
+---
+
+Added new container ghost charactersitic tokens
```

Comment:
typo

Reply:
will see whether can fix in another pr. chromatic / test cycle is too long